nengo_loihi/tests/test_connection.py

```diff
@@ -749,3 +749,21 @@ def test_input_synapses(Simulator, allclose, plt):
 
     # Only looking at t < 0.4 as there are weird effects at the end
     assert allclose(ref_filt[t < 0.4], sim_filt[t < 0.4], atol=1.5)
+
+
+@pytest.mark.skipif(nengo_transforms is None, reason="Requires new nengo.transforms")
+def test_sparse_transforms(Simulator):
```

Comment:
Maybe `test_sparse_transform_empty_neuron` or something like that, since we're specifically testing sparse transforms that have some neurons that get no inputs, not sparse transforms generally. Also, we should put this description in a little docstring.